tensorflow/compiler/xrt/xrt_memory_manager.cc

```diff
@@ -265,8 +268,8 @@ void XRTMemoryManager::ReleaseAllAllocations() {
 }
 
 xla::StatusOr<se::OwningDeviceMemory> XRTMemoryManager::Allocate(
-    xla::Backend* backend, int device_ordinal, size_t size) {
-  se::DeviceMemoryAllocator* allocator = backend->memory_allocator();
+    xla::Backend* backend, int device_ordinal, size_t size,
+    se::DeviceMemoryAllocator* allocator) {
```

Comment:
do we need allocator here?

Reply:
Yes, because we need to call `allocator->Allocate` in this function.